Orange/misc/server_embedder.py

```diff
@@ -111,25 +118,25 @@ def embedd_data(
         EmbeddingCancelledException:
             If cancelled attribute is set to True (default=False).
         """
-        # if there is less items than 10 connection error should be raised
-        # earlier
+        # if there is less items than 10 connection error should be raised earlier
         self.max_errors = min(len(data) * self.MAX_REPEATS, 10)
 
         loop = asyncio.new_event_loop()
         asyncio.set_event_loop(loop)
         try:
             embeddings = asyncio.get_event_loop().run_until_complete(
-                self.embedd_batch(data, processed_callback)
+                self.embedd_batch(data, processed_callback, callback)
             )
-        except Exception:
+        finally:
             loop.close()
-            raise
 
-        loop.close()
         return embeddings
 
     async def embedd_batch(
-            self, data: List[Any], proc_callback: Callable[[bool], None] = None
+        self,
+        data: List[Any],
+        processed_calback: Optional[Callable] = None,
```

Comment:
Typo. :) Now I see that you mistyped both uses so the thing actually works!

Why are you renaming it? Renaming makes calls as a kwarg (previously valid) invalid.

Reply:
It was a mistake since first I wanted to fix it one way and then I changed back.